pkg/vm/vm_linux.go

```diff
@@ -131,6 +131,21 @@ func (v *BootcVMLinux) Run(params RunVMParameters) (err error) {
 		}
 	}
 
+	if v.domain != nil {
+		isRunning, err := v.IsRunning()
+		if err != nil {
+			return fmt.Errorf("unable to check if VM is running: %w", err)
+		}
+
+		if !isRunning {
+			logrus.Debugf("Deleting stopped VM %s\n", v.imageID)
+			v.Delete()
```

Comment:
you forgto your last changes from gitlab:
```
			logrus.Debugf("Deleting stopped VM %s\n", v.imageID)			
			if err := v.Delete(); err != nil {
				return fmt.Errorf("unable to delete stopped VM: %w", err)
			}
```

Reply:
whoops - good catch, I guess I made that change from my mac. Updated.